core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -476,6 +478,7 @@ object KafkaConfig {
   val ReplicaFetchWaitMaxMsDoc = "max wait time for each fetcher request issued by follower replicas. This value should always be less than the " +
   "replica.lag.time.max.ms at all times to prevent frequent shrinking of ISR for low throughput topics"
   val ReplicaFetchMinBytesDoc = "Minimum bytes expected for each fetch response. If not enough bytes, wait up to replicaMaxWaitTimeMs"
+  val ReplicaFetchResponseMaxBytesDoc = "Maximum bytes expected for entire fetch respones."
```

Comment:
`respones` should be `response`. Also, it's probably a good idea to clarify the doc for `ReplicaFetchMaxBytesDoc` as it's a bit vague.